client/lib/signup/step-actions/passwordless.js

```diff
@@ -9,11 +11,16 @@ import wpcom from 'calypso/lib/wp';
  * @param {string}   data.email
  */
 export function createPasswordlessUser( callback, { email } ) {
-	wpcom
-		.undocumented()
-		.usersEmailNew( { email }, null )
-		.then( ( response ) => callback( null, response ) )
-		.catch( ( err ) => callback( err ) );
+	wpcom.req.post(
+		'/users/email/new',
+		{
+			email,
+			locale: getLocaleSlug(),
```

Comment:
Is the `locale` necessary here? I don't see it in the old `usersEmailNew` method.

Reply:
It's true that it wasn't there before, but all endpoints that create a new user should have the `locale` param. The locale of the new user is set accordingly. The various user creation endpoints do this inconsistently, some don't honor the `locale` param. This `/users/email/new` seems to be an example of that. But the client should include it.

Reply:
Gotcha 👍  Makes sense!